content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/README.md

```diff
@@ -106,6 +107,10 @@ CSS styling. It should be added to a relevant site client library using the `emb
 It also provides a `core.wcm.components.image.v1.editor` editor client library category that includes
 JavaScript handling for dialog interaction. It is already included by its edit dialog.
 
+## SVG
+SVG MIME-types are supported, but have some specific handling. Alternative image sizes defined in the component policy dialog are ignored for SVG images. In addition, SVG image types have a more limited set of
+editing options available in the AEM inline image editor.
```

Comment:
I would be more specific and state that smart sizes are not supported and that Image#getWidths returns an empty array.
We should also add that lazy loading is supported.

Reply:
Thanks @jckautzmann - updated in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/200/commits/6d18fcfe113db5d00a26670dcde8b684613e1a63